scene/resources/visual_shader_nodes.h

```diff
@@ -1486,6 +1486,8 @@ class VisualShaderNodeFloatUniform : public VisualShaderNodeUniform {
 	float hint_range_min;
 	float hint_range_max;
 	float hint_range_step;
+	bool default_value_enabled;
```

Comment:
This should likely be done in a separate PR, but we now initialize member variables directly in their declaration in the header, instead of deferring it to the constructor.

So you could refactor all VisualShaderNodes to move all those initializations to the header.